source/index.md

```diff
@@ -2,9 +2,12 @@
 
 - [Atlas Serverless Tests](serverless-testing/README.md)
 - [Authentication](auth/auth.md)
-- [BSON Binary Subtype 6](client-side-encryption/subtype6.md)
+- [BSON Binary Encrypted](bson-binary-encrypted/binary-encrypted.md)
```

Comment:
Similar to how we have multiple documents in the SDAM spec folder, how about moving all of the various BSON specs into a bson folder with an index document that links to the main BSON spec (https://bsonspec.org) and our own specifications?

Reply:
Works for me 👍

Reply:
Haha, I meant that as a question.  Should I make a `source/bson` folder and put the other `bson-` specs in it now?

Reply:
Oh, sorry. Yes, having a `bson` folder containing all of the various BSON specs makes sense, I'm also fine leaving the `bson-corpus` out for the time being until we know all of the references.

There could also be a single `tests` folder that contains the various tests (potentially in subfolders similar to how it's done for the CSFLE spec). Looking at #1658, the question is why those tests aren't added to the BSON corpus in the first place - was it a matter of grouping tests with the spec document?

Reply:
The tests are different from the standard corpus tests, because there is additional metadata in the BSON binary value.  So I think we should keep these folders separate.